### PR TITLE
[DOC] ColumnExpectation, render_evaluation_parameter_string and validate method

### PR DESCRIPTION
**Pull request by josectobar**
Changes proposed in this pull request:
- Documentation for ColumnExpectation
- Documentation for render_evaluation_parameter_string
- Documentation for validate

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def validate_configuration(
             configuration: The ExpectationConfiguration to be validated.
 
         Raises:
-            InvalidExpectationConfigurationError: The configuraton does not contain the values required by the
+            InvalidExpectationConfigurationError: The configuration does not contain the values required by the
```
Comment (Shinnnyshinshin) [inline great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py:350]: 🙇🏼 thanks for catching this

File: great_expectations/expectations/expectation.py [hunk 2]
```diff
@@ -143,7 +143,20 @@
 )
 
 
+@public_api
 def render_evaluation_parameter_string(render_func) -> Callable:
+    """Decorator for Expectation classes that renders evaluation parameters as strings.
```
Comment (Shinnnyshinshin) [inline great_expectations/expectations/expectation.py:148]: Flagging this function. `render_evaluation_parameter_string` does not seem to be rendering in the Netlify preview. cc @anthonyburdi

File: great_expectations/expectations/expectation.py [hunk 3]
```diff
@@ -2493,7 +2522,30 @@ def validate_configuration(
             warnings.warn(str(e), UserWarning)
 
 
+@public_api
 class ColumnExpectation(TableExpectation, ABC):
+    """Base class for column-type Expectations.
+
+    These types of Expectation produce an aggregate metric for a column, such as the mean, standard deviation,
```
Comment (Shinnnyshinshin) [inline great_expectations/expectations/expectation.py:2529]: 👍🏼

File: great_expectations/expectations/expectation.py (hunk 2 above)
Comment (anthonyburdi) [reply great_expectations/expectations/expectation.py:148]: Looks like it is working now: https://deploy-preview-6995--niobium-lead-7998.netlify.app/docs/reference/api/expectations/expectation#great_expectations.expectations.expectation.render_evaluation_parameter_string

Comment (Shinnnyshinshin) [reply great_expectations/expectations/expectation.py:148]: whoohoo!

**Review by Shinnnyshinshin — APPROVED** [review]
🚀

**Pull request merged**